Gemfile.lock

```diff
@@ -9,20 +9,20 @@ GIT
 GEM
   remote: https://rubygems.org/
   specs:
-    appraisal (2.1.0)
+    appraisal (2.2.0)
```

Comment:
Do you mind breaking these changes to the gemfiles off into a separate commit? It looks like the crux of these changes is upgrading Appraisal from 2.1.0 to 2.2.0.